app/src/util/highlightWord.js

```diff
@@ -0,0 +1,6 @@
+export default (strReplace, str, styleClass) => {
```

Comment:
Do we really need this function to be apart in its separate file? If it's used in just one component, let's put it in the component.